tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/generators/cpp/CppClientGenerator.java

```diff
@@ -125,6 +123,33 @@ public SdkFileEntry[] generateSourceFiles(ServiceModel serviceModel) throws Exce
         return fileList.toArray(retArray);
     }
 
+    protected void addEventStreamInitialResponse(final ServiceModel serviceModel) {
+        for (Map.Entry<String, Operation> entry : serviceModel.getOperations().entrySet()) {
```

Comment:
nit: you know I love java streams so optional streams refactor:
```
 serviceModel.getOperations().entrySet().stream()
  .filter(operation -> Objects.nonNull(operation.getValue().getResult()))
  .filter(operation -> operation.getValue().getResult().hasEventStreamMembers())
  .map(operation -> Shape.builder()
                .name(operation.getKey() + "InitialResponse")
                .type("structure")
                .isReferenced(true)
                .event(true)
                .eventPayloadType("structure")
                .members(operation.getValue().getResult().getMembers().entrySet().stream()          
                    .filter(member -> !member.getValue().getShape().isEventStream())
                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)))
                .build();)
  .foreach(shape -> serviceModel.getShapes().put(sahpe.getName+  "InitialResponse", shape); )
```

(i didnt check if that compiles as is, but it should be close if you want to try to use more streams)